core/src/main/resources/com/github/weisj/darklaf/ui/bundles/tabFrame.properties

```diff
@@ -22,11 +22,11 @@
 #
 # suppress inspection "UnusedProperty" for whole file
 popup.moveTo            = Move to
-popup.moveTo.north      = Top Left
-popup.moveTo.south      = Bottom Right
-popup.moveTo.east       = Right Top
-popup.moveTo.west       = Left Bottom
+popup.moveTo.north      = Top
+popup.moveTo.south      = Bottom
+popup.moveTo.east       = Right
+popup.moveTo.west       = Left
 popup.moveTo.north_east = Top Right
-popup.moveTo.north_west = Left Top
-popup.moveTo.south_east = Right Bottom
+popup.moveTo.north_west = Top Left
+popup.moveTo.south_east = Bottom Right
 popup.moveTo.south_west = Bottom Left
```

Comment:
It may seem unintuitive at first, but they way they are currently is actually how it should be. There are 8 total directions positioned as follows (they are from a custom component, which replicates the frame from IntelliJ):

````
     1       |     2
---+---------+--------+---
   |                  |
8  |                  | 3
   |                  | 
---+                  +---
   |                  |
 7 |                  | 4
   |                  |
---+---------+--------+---
     6       |     5
````
1: north
2: north_east
3: east
4: south_east
5: south
6: south_west
7: west
8: north_west

Note that for e.g. 1 and 8 the wording needs to be in different orders to distinguish them.

Reply:
this is crazy